configuration/index.md

```diff
@@ -128,14 +129,39 @@ Every interface has it's own strong points.
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.X, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI". It allows for configuration of the base system and setup of openHAB 2
+addons. Please be aware that not every addon/binding that has been existing since openHAB version 1 has been updated to version 2 and
```

Comment:
- Add line break after "addons."
- addons -> add-ons
- that has been existing since openHAB version 1 -> of openHAB 1.x
- remove line break at the end